src/core/Navigation/SideNavigation/SideNavigation/SideNavigation.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { forwardRef, ReactNode, useState } from 'react';
+import {
+  HtmlButton,
+  HtmlDiv,
+  HtmlNav,
+  HtmlSpan,
+  HtmlUl,
+} from '../../../../reset';
+import styled from 'styled-components';
+import { SuomifiThemeConsumer, SuomifiThemeProp } from '../../../theme';
+import { baseStyles } from './SideNavigation.baseStyles';
+import classnames from 'classnames';
+import { Icon } from '../../../Icon/Icon';
+import {
+  StaticIcon,
+  IllustrativeIconKeys,
+  DoctypeIconKeys,
+} from '../../../StaticIcon/StaticIcon';
+import { getConditionalAriaProp } from '../../../../utils/aria';
+
+export interface SideNavigationProps {
+  /** Use the `<SideNavigationItem>` components as children */
+  children: ReactNode;
+  /** Name for the navigation element. Don't use the word "navigation" since it will be read by screen reader regardless. */
+  'aria-label': string;
+  /** HTML nav element will receive this id */
```

Comment:
Just a style difference compared to the rest of the library, but caught my eye. Could this be "id passed to the HTML nav element" or "id for the HTML nav element"?

Reply:
Good point. Changed it!